### DISCUSSION TITLE
Compatability with new Google Play Subscriptions API

### DISCUSSION THREAD

**Discussion by SimonDamberg** [hyochan/react-native-iap#1709] [Q&A] [answered]
Hello!

How does this library handle the new [Google Play Subscriptions API](https://support.google.com/googleplay/android-developer/answer/12124625) that has replaced the old one? For example, montly and yearly subscriptions now share the same product-id but are listed as different "base-plans" in that subscription. When I fetch the product-id for that subscription, I only get one of the plans.

**Comment by andresesfm** [comment, marked as answer]
v 9.0.0 released
